var/spack/repos/builtin/packages/py-ipdb/package.py

```diff
@@ -26,8 +31,8 @@ class PyIpdb(PythonPackage):
     # However additional dependencies added below were found via testing.
```

Comment:
python versions above are incorrect. Honestly, you could just remove that line, Spack only supports 3.7+ anyway